app/assets/stylesheets/components/_password.scss

```diff
@@ -1,39 +1,34 @@
 @use 'uswds-core' as *;
 
-$weak: #e80e0e;
-$average: #ffac00;
-$good: #9ac056;
-$great: #00b200;
-
-.pw-bar {
-  background-color: #e9e9e9;
-  border: units(0.5) solid #fff;
-  border-radius: 6px;
-  float: left;
-  height: 16px;
-  width: 25%;
+.password-strength__meter {
+  display: flex;
+  margin-top: units(1);
+  margin-bottom: units(0.5);
 }
 
-.pw-weak {
-  .pw-bar:nth-child(-n + 1) {
-    background-color: $weak;
+.password-strength__meter-bar {
+  flex-basis: 25%;
+  background-color: color('base-lighter');
+  border-radius: 2px;
+  height: units(1);
+
+  & + & {
+    margin-left: units(1);
   }
-}
 
-.pw-average {
-  .pw-bar:nth-child(-n + 2) {
-    background-color: $average;
+  .pw-weak &:nth-child(-n + 1) {
```

Comment:
looks like `pw-weak` , etc are defined in `packs/pw-strength.js`, should we update that to use the expanded `password-strength__weak` style or just leave it?

Reply:
Yep, I've got a lot more planned here for follow-on pull requests, and this is one of them. The problem I've had with password strength is that every time I've tried to touch it in the past, I end up doing a major overhaul, so I'm trying to do this a little more incrementally this time 😅